scripts-available/CDB_Organizations.sql

```diff
@@ -104,14 +104,38 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE PARALLEL UNSAFE;
 
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Get_Table_Sequences(from_schema text, table_name text)
+    RETURNS SETOF TEXT
+AS $$
+BEGIN
+    RETURN QUERY EXECUTE 'SELECT
+        n.nspname || ''.'' || c.relname
+    FROM
+        pg_depend d
+        JOIN pg_class c ON d.objid = c.oid
+        JOIN pg_namespace n ON c.relnamespace = n.oid
+    WHERE
+        d.refobjsubid > 0 AND
+        d.classid = ''pg_class''::regclass AND
+        c.relkind = ''S''::"char" AND
+        d.refobjid = (''' || from_schema || '.' || table_name ||''')::regclass';
```

Comment:
Namely: `.refobjid = (''"' || from_schema || '"."' || table_name ||'"'')::regclass';`
😅 wow, that's hard to read
(this would avoid problems with spaces, dashes, etc. in the names)
And be thankful I'm not getting into supporting names with quotes or doubles quotes in them 😛

Reply:
I did add them in a commit but then reverted it after looking at documentation. Since this is already inside a string (generated SQL looks like `refobjid = ('schema-user.table-name')::regclass`), there is no need to wrap in double quotes (this is only needed when using the name as an identifier) for dashes.

However, fair is fair and this breaks for spaces, quotes, etc (which we don't support, but better to be safe than sorry). I'm adding `quote_ident` since that should take care of all problems, and it's better than adding partial fixes.